cypress/e2e/common/common.js

```diff
@@ -996,6 +995,85 @@ Given("I seeded the database",() => {
   })
 })
 
+Given("I seeded the database with an asset {string} with no attached tenure", (assetGuid) => {
+  cy.log("Seeding database").then(() => {
+    const patchModel = patch;
+    const assetModel = getAssetWithNoTenure(assetGuid, patchModel)
+    const personModel1 = person();
+    const personModel2 = person();
+    const tenureModel = tenure({}, assetModel, [personModel1, { isResponsible: true, personTenureType: "Tenant", ...personModel2 }]);
+
+    return new Cypress.Promise((resolve) => {
+      Promise.all([
+        DynamoDb.createRecord("PatchesAndAreas", patchModel),
+        DynamoDb.createRecord("Assets", assetModel),
+        DynamoDb.createRecord("Persons", personModel1),
+        DynamoDb.createRecord("Persons", personModel2),
+      ]).then(() => {
+        resolve()
+      })
+    }).then(() => {
+      cy.log("Database seeded!");
+    })
+  })
+})
+
+Given("I seeded the database with an asset {string} with a previous tenure", (assetGuid) => {
```

Comment:
Used to verify that an error is displayed when trying to start a new tenure with a start date that's earlier than the previous tenure's end date.